.claude/skills/cmux-codex-autoapprove/scripts/cmux_codex_approve_launchd.py

```
@@ -0,0 +1,738 @@
+                    cooldown = 0.0
+                state[key] = {"digest": digest, "cooldown_until": cooldown}
+    return state
+
+
+def save_state(state: dict[str, dict[str, float | str]]) -> None:
+    STATE_FILE.write_text(json.dumps(state, sort_keys=True), encoding="utf-8")
+
+
+def run(cmd: list[str], **kwargs) -> subprocess.CompletedProcess[str]:
+    env = kwargs.pop("env", None)
+    return subprocess.run(cmd, text=True, capture_output=True, env=env, **kwargs)
```

Ruff (0.15.9)

[error] `subprocess` call: check for execution of untrusted input

(S603)